tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -98,6 +98,65 @@ def test_positive_create_rhev_with_name(self):
                     self.assertIsNotNone(self.compute_resource.search(name))
 
     @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1467664)
+    @tier1
+    def test_positive_create_rhev_with_existing_name(self):
```

Comment:
Maybe we should think about more descriptive name, this one is a bit confusing - combination of 'positive' and 'create_with_same_name' sound a bit weird :) I don't have better ideas though.
Oh, and `s/existing/same/`, all existing tests use that word.